src/main/java/org/galatea/starter/AppConfig.java

```diff
@@ -38,6 +43,29 @@ public IAgreementTransformer agreementTransformer() {
         .direction("B".equals(agreement.getBuySell()) ? "REC" : "DEL").version(0L).build();
   }
 
+  /**
+   * CacheManager that contains the Cache Configuration. Spring will use this cache if any
+   * cache annotations are used.
+   */
+  @Bean
+  public CacheManager cacheManager(final EhCacheManagerFactoryBean ehCacheCacheManager) {
+    return new EhCacheCacheManager(ehCacheCacheManager.getObject());
+  }
+
+  /**
+   * Creates factory bean for cache manager using a cache config file.
+   *
+   * @return factory bean for the EhCache to be passed to EhCacheCacheManager.
+   */
+  @Bean
+  public EhCacheManagerFactoryBean ehCacheCacheManager(
```

Comment:
Can we change the name of this bean so that it's clear that it isn't itself the cache manager?

Reply:
I've updated the name of the bean and the parameter in the bean above.